frontend/internal-packages/agent/src/chat/workflow/README.md

```diff
@@ -7,17 +7,26 @@ A **LangGraph implementation** for processing chat messages in the LIAM applicat
 ```mermaid
 flowchart TD
     START([START])
-    A[validationNode<br/>Input Validation]
-    B[answerGenerationNode<br/>Answer Generation]
-    C[finalResponseNode<br/>Final Response]
-    END([END])
+    PM[pmAgent<br/>Requirements Organization]
+    DB[dbAgent<br/>DB Design & DDL Execution]
+    QA[qaAgent<br/>Use Case Verification & DML Execution]
+    PM_CHECK[pmAgentReview<br/>Final Requirements & Deliverables Confirmation]
+    ARTIFACT[dbAgentArtifactGen<br/>Artifact Generation]
+    SAVE[saveToDB<br/>Save Results]
+    FORMAT[finalResponse<br/>Final Response Formatting]
+    END([__end__<br/>End])
+
+    START --> PM
+    PM --> DB
+    DB --> QA
+    QA -->|success| PM_CHECK
+    QA -->|dml error or test fail| DB
+    PM_CHECK -->|OK| ARTIFACT
+    PM_CHECK -->|NG or issues found| PM
+    ARTIFACT --> SAVE
+    SAVE --> FORMAT
+    FORMAT --> END
 
-    START --> A
-    A -->|success| B
-    A -->|error| END
-    B -->|success| C
-    B -->|error| END
-    C --> END
 ```
 
 ## Workflow State
```

Comment:
I think the WorkflowState will need updates for the new multi-agent architecture.